ogusa/tests/test_run_example.py

```diff
@@ -0,0 +1,43 @@
+#### import modules (same as run_ogusa_example)
+import multiprocessing
+from distributed import Client
+import time
+import numpy as np
+import os, sys
+import taxcalc
+from taxcalc import Calculator
+from ogusa import output_tables as ot
+from ogusa import output_plots as op
+from ogusa.execute import runner
+from ogusa.constants import REFORM_DIR, BASELINE_DIR
+from ogusa.utils import safe_read_pickle
+
+currentdir = os.path.dirname(os.path.realpath(__file__))
+parentdir = os.path.dirname(currentdir)
+sys.path.append(parentdir)
```

Comment:
@jpycroft  and @jdebacker. We probably need to use a different method of loading this module than the `sys.path.append`. It is discouraged to amend the path like this. I think that every time this is called, this will add a string to the `sys.path` list argument. But I have spent the last three hours trying to get a way and none work yet.

Reply:
@rickecon and @jdebacker. Point taken Rick. 

Let me share an alternative solution, which might help. The code is here: 
https://github.com/jpycroft/OG-USA/blob/jptest/ogusa/tests/test_run_example-wi-pathlib.py

It does work, though I'd say it's less elegant, as you need to change run_ogusa_example.py.
Using pathlib, I navigate to and execute the whole file "run_ogusa_example.py". 
However, the "if __name__ == "__main__":" condition fails, so you have to add another condition:
    if __name__ == "test_run_example-wi-pathlib":
        main()

I haven't _yet_ found a way of using pathlib to be able to call run_ogusa_example.main() directly. I will keep looking though!

Reply:
@rickecon and @jdebacker. I've uploaded a new test_run_example file. 

It uses importlib.util in place of sys.path.append to locate the module. It also avoids any need to alter run_ogusa_example.py in any way (see previous comment).